common/test/acceptance/pages/studio/users.py

```diff
@@ -92,6 +136,30 @@ def url(self):
         return "{}/library/{}/team/".format(BASE_URL, unicode(self.locator))
 
 
+class CourseTeamPage(CoursePage, UsersPageMixin):
+    """
+    Course Team page in Studio.
+    """
+
+    url_path = "course_team"
+
+    def wait_until_ready(self):
+        """
+        When the page first loads, there is a loading indicator and most
+        functionality is not yet available. This waits for that loading to
+        finish.
+
+        Always call this before using the page. It also disables animations
+        for improved test reliability.
+        """
+        self.wait_for_ajax()
+        self.wait_for_element_invisibility(
+            '.ui-loading',
+            'Wait for the page to complete its initial loading of XBlocks via AJAX'
```

Comment:
XBlocks? On the Team Course page? Are we sure this wait_until_ready function is doing what we intend for it to?